features/core-verify-checksums.feature

```diff
@@ -63,3 +63,19 @@ Feature: Validate checksums for WordPress install
       """
       Success: WordPress install verifies against checksums.
       """
+
+  Scenario: Verify core checksums with extra files
+    Given a WP install
+
+    When I run `wp core update`
+    Then STDOUT should not be empty
+
+    When I run `touch wp-includes/extra-file.txt`
```

Comment:
@danielbachhuber Not sure why this isn't working. Any idea how to create an extra file to test this with?